common/common.go

```diff
@@ -22,3 +22,10 @@ const (
 	// DisableDefaultInstallEnvVar is an env variable to disable the default instance
 	DisableDefaultInstallEnvVar = "DISABLE_DEFAULT_ARGOCD_INSTANCE"
 )
+
+// InfraNodeSelector is openshift label for infrastructure nodes since maps are not constants returning from a funciton
+func InfraNodeSelector() map[string]string {
+	return map[string]string{
+		"node-role.kubernetes.io/infra": "",
+	}
+}
```

Comment:
@saumeya We probably don't need a new function just for this, what I meant earlier was just a suggestion that we could define a new constant in this file,something like  `InfraNodeLabel = "node-role.kubernetes.io/infra" `

And use it wherever needed as ```common.InfraNodeLabel: "" ``` (such as [here](https://github.com/redhat-developer/gitops-operator/pull/215/files#diff-83cb198211fecd2e48f050de2a80a348f8525ade03f84535fbe925fb8dc3608aR328) )

The idea was just that in case the label changes in the future we can just make the change in this one place rather than having to change it 5-6 times if we hard code the string in multiple places

Reply:
oh that wont work, becasue nodeSelector is a map not a string

Reply:
got it Jaideep, but adding a string constant in common file won't decrease the size of code at all, i'll still have to instantiate a map everytime. Unless having a common function is incorrect for the purpose of common file, can this part remain?